### ISSUE TITLE
Particular PMID lookup fails

### ISSUE THREAD

**Issue by kermitt2** [kermitt2/biblio-glutton#31]
http://localhost:8080/service/lookup?pmid=16262981 results in 404

However it is the PMID indicated by this record, so it exists ! 
http://localhost:8080/service/lookup?pii=S0266462305050762

**Comment by lfoppiano** [comment]
it looks not in the PMID source file: 

```
(base) Johan:consolidationData lfoppiano$ zgrep 16262981 PMID_PMCID_DOI.csv.gz 
(base) Johan:consolidationData lfoppiano$ 
```

**Comment by kermitt2** [comment]
but present in the ISTEX ids file

> lopez@work:/mnt/data/biblio$ zgrep 16262981 ~/biblio-glutton/data/istex/istexIds.all.gz 
{"corpusName":"cambridge","istexId":"CC91E0F1789978CE79D653533100BA315CA337B3","ark":["ark:/67375/6GQ-9RTTRBZ7-G"],"doi":["10.1017/S0266462305050762"],"pmid":["16262981"],"pii":["S0266462305050762"]}

**Comment by lfoppiano** [comment]
right... one thing I notice is that we do not use the pmid information from the istex mapping but only from the pmid file (which should be the authority here, isn't it?). 
We should complement the pmid lookup with information from istex mapping, perhaps?

**Comment by kermitt2** [comment]
Yes I think it makes sense to complement like that the pmid lookup. 

The main issue is then that the publisher metadata are not well maintained. When a PMID becomes invalid at PubMed (which is the reference), apparently they are not removed from the publisher metadata and we would have invalid mapping via the istex data file.

The submodule `pubmed-glutton` map DOI/PMID to MESH classes using the officiel PubMed metadata dump files. One approach to control the pmid information would be to exploit this additional mapping. I still need then to work a bit on it again. 

So I would suggest to wait until I review the `pubmed-glutton` submodule. This module will produce an additional mapping PMID/MESH classes to enrich the record of biblio-glutton, and we could use it to control valid PMID.
